src/main/java/uk/gov/hmcts/reform/unspec/launchdarkly/OnBoardingOrganisationControlService.java

```diff
@@ -0,0 +1,69 @@
+package uk.gov.hmcts.reform.unspec.launchdarkly;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.launchdarkly.sdk.LDValue;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.prd.model.Organisation;
+import uk.gov.hmcts.reform.unspec.service.OrganisationService;
+import uk.gov.hmcts.reform.unspec.service.UserService;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class OnBoardingOrganisationControlService {
+
+    public static final String ORG_NOT_REGISTERED = "your organisation is %s, unfortunately that org "
```

Comment:
is this the message we want to display to the user?

Reply:
yes

Reply:
Are you 100% sure? I see it says `You cannot use this service because your organisation is not part of the HMCTS civil damages pilot.` in ACs.

Reply:
I would chase up Steve for content